src/testSetup.ts

```diff
@@ -304,6 +305,41 @@ export class TestContext {
     this.configStubs.AuthInfoConfig = { retrieveContents };
   }
 
+  /**
+   * Stub salesforce user authorizations.
+   *
+   * @param users The users to stub.
+   * The key is the username of the admin user and it must be included in the users array in order to obtain the orgId key for the remaining users.
+   * The admin user is excluded from the users array.
+   *
+   */
+  public stubUsers(users: Record<string, MockTestOrgData[]>): Promise<void> {
```

Comment:
should this be async? or just return `void`?

Reply:
Goes along with others, although no awaits

Reply:
why follow the others if there's no `await`s? will you have to `await stubUsers` since it returns a promise?